proof/invariant-abstract/ARM/ArchCSpace_AI.thy

```diff
@@ -604,8 +604,6 @@ lemma arch_post_cap_deletion_invs:
   "\<lbrace>invs and (\<lambda>s. arch_post_cap_delete_pre (ArchObjectCap c) (caps_of_state s))\<rbrace> arch_post_cap_deletion c \<lbrace>\<lambda>rv. invs\<rbrace>"
   by (wpsimp simp: arch_post_cap_delete_pre_def)
 
-end
-
 (* is this the right way? we need this fact globally but it's proven with
    ARM defns. *)
```

Comment:
Very minor, but it looks like you've removed this comment on most architectures but left it for this one. Was that intentional?

Reply:
nope, thanks